docs/whatsnew/1.0.0/1_0_0.rst

```diff
@@ -0,0 +1,81 @@
+Pylake 1.0.0
+============
+
+.. only:: html
+
+Pylake has made it to `v1.0.0`! And with it, several new features and improvements have been added.
+
+Documentation
+-------------
+
+.. image:: docs.png
+
+The documentation has been completely reworked.
+There is now a new :doc:`theory section</theory/index>` which delves into the theory of the underlying data analysis methods, including examples and references to literature for further reading.
+This information has been separated from the :doc:`tutorial section</tutorial/index>` so the tutorials can focus specifically on teaching you how to use Pylake.
+In addition, both sections were extended considerably.
+
+Every :doc:`tutorial</tutorial/index>` now contains links to example datasets hosted on `Zenodo <https://zenodo.org/>` that are downloaded directly when running the Jupyter notebooks.
+This allows you to immediately try out the various functionalities Pylake has to offer.
```

Comment:
~~~suggestion
Every :doc:`tutorial</tutorial/index>` now contains links to real C-Trap datasets hosted on `Zenodo <https://zenodo.org/>`. They are downloaded as part of running the Jupyter notebooks.
It simplifies trying out all the available Pylake functionality with realistic use-cases provided by the Pylake team.
~~~

Reply:
👍